flow/envs/test.py

```diff
@@ -52,3 +52,17 @@ def compute_reward(self, rl_actions, **kwargs):
     def get_state(self, **kwargs):
         """See class definition."""
         return np.array([])
+
+class TestI210Env(TestEnv):
+
+    def additional_command(self):
+        edge = "119257908#0"
+        edge_length = self.k.network.edge_length(edge)
+        for veh_id in self.k.vehicle.get_ids():
+            edge = self.k.vehicle.get_edge(veh_id)
```

Comment:
Need new variable for this lil guy or there will be a name conflict and then line 66 makes no sense

Reply:
Deleted.